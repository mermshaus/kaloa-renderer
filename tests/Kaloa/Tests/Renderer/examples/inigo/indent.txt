[indent]foo

bar

    [indent][h3]baz & boz[/h3][/indent]
[/indent]

[center]* * *[/center]

Foo