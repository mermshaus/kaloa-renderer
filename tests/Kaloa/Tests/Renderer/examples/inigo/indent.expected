<div style="margin-left: 30px;"><p>foo</p>

<p>bar</p>
<div style="margin-left: 30px;"><h3>baz &amp; boz</h3>

</div>

</div>

<div style="text-align: center;">* * *</div>

<p>Foo</p>
